main.py

```diff
@@ -1,7 +1,11 @@
+from rules import rules
 from draw_ball import ball
 
+
 def main():
-    print (ball(5))
+    random = ball(5)
```

Comment:
Please do not use "random" as a variable name. First because `random` is a library from the python standard library, and it could be misleading.
Second, it is hard to understand what type of object is your "random"